pl_examples/basic_examples/lightning_module_template.py

```diff
@@ -16,9 +16,10 @@
 from torchvision.datasets import MNIST
 
 import pytorch_lightning as pl
```

Comment:
then this is not used at all, right?

Reply:
@williamFalcon this I was proposing some time ago, but you didn't like it lol

Reply:
> @williamFalcon this I was proposing some time ago, but you didn't like it lol

oh, that's not true. It was the one of the very first issue and the warning said that it can't find LightningModule. That's why I have to refer its full name directly. Let me see if I can stick to the convention.

Reply:
> then this is not used at all, right?

Removed. 886eb264b7ec0e5686afa7cfee5942e942b31f37

Reply:
> @williamFalcon this I was proposing some time ago, but you didn't like it lol

I found a much cleaner way to do it, sticking to the previous convention. 4287eb0d5bf3da3a499aab0c275bc474892f3bc9